docs/topics/jvm/jvm-get-started.md

```diff
@@ -4,23 +4,33 @@ This tutorial demonstrates how to use IntelliJ IDEA for creating a console appli
 
 To get started, first download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
 
-## Create an application 
-
-Once you've installed IntelliJ IDEA, it's time to create your first Kotlin application.
+## Create a project
 
 1. In IntelliJ IDEA, select **File** | **New** | **Project**.
-2. In the panel on the left, select **Kotlin**.
-3. Enter a project name, select **Console Application** as the project template, and click **Next**.
-   
-   ![Create a console application](jvm-new-project-1.png){width=700}
+2. In the panel on the left, select **New Project**.
+3. Name the new project and change its location if necessary.
+4. Select the **Create Git repository** checkbox to place the new project under version control. You will be able to do it
+later at any time.
+5. From the **Language** list, select **Kotlin**.
    
-   By default, your project will use the Gradle build system with Kotlin DSL.
+   ![Create a console application](jvm-new-project.png){width=700}
+
+6. Select the **IntelliJ** build system. It's a native builder that doesn't require downloading additional artifacts.
```

Comment:
Why IntelliJ?
Usually we don't promote it

Reply:
I don't think it's useful when we explain how to start a project with IntelliJ build system and then everywhere talk only about the Gradle/Maven.
What do you think?